scripts/psycorona_modelling.R

```diff
@@ -69,6 +73,77 @@ preds        <- names(df_analyse)[-(which(names(df_analyse) %in% outcome_vars))]
 # Explore outcome variable
 hist(df_analyse$c19perbeh)  # people tend to self-serve
 
+
+########## CARE MODEL COMPARING SETUP ########## 
+
+# make the CV split
+cv_split <- trainControl(method="cv", number=10, verboseIter=TRUE)
+# Train the LASSO 
+lambda <- seq(0.0001,0.01,by = 0.0001)
```

Comment:
@AndriiG13 what is this sequence based on?

Reply:
This is based on a couple of trials I ran. I started off with sequences of larger lambdas and eventually settled on this, as the optimal lambda I get with this seed is .0006. 
We could always get more fine grained, but my goal with the commit was more to setup the caret framework than to optimise the models